mobile_verifier/src/reward_shares.rs

```diff
@@ -318,22 +457,24 @@ mod test {
     use super::*;
     use crate::{
         cell_type::CellType,
+        coverage::{CoveredHexStream, HexCoverage, Seniority},
         data_session,
         data_session::HotspotDataSession,
-        heartbeats::{HeartbeatReward, HeartbeatRow},
+        heartbeats::{HeartbeatReward, HeartbeatRow, KeyType, OwnedKeyType},
         speedtests::Speedtest,
+        speedtests_average::SpeedtestAverage,
         subscriber_location::SubscriberValidatedLocations,
     };
     use chrono::{Duration, Utc};
     use file_store::speedtest::CellSpeedtest;
-    use futures::stream;
+    use futures::stream::{self, BoxStream};
     use helium_proto::services::poc_mobile::mobile_reward_share::Reward as MobileReward;
     use prost::Message;
     use std::collections::HashMap;
     use uuid::Uuid;
 
     #[test]
-    fn bytes_to_bones() {
+    fn ensure_correct_conversion_of_bytes_to_bones() {
```

Comment:
weird naming choice, do we ever use an incorrect conversion?

Reply:
The ensure? This just properly describes what the test does